statvar_imports/southkorea_statistics/education/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "import_specifications": [
+    {
+      "import_name": "SouthKorea_Education",
+      "curator_emails": ["support@datacommons.org"],
+      "provenance_url": "https://kosis.kr/eng/statisticsList/statisticsListIndex.do",
+      "provenance_description": "This data is from the South Korea Statistics Agency (KOSA), that describes about the education statistics of South Korea",
+      "scripts": ["../run.sh gs://unresolved_mcf/country/southkorea/education/input_files/",
+        "../../../tools/statvar_importer/stat_var_processor.py --input_data=source_files/Summary_of_Elementary_School_20250821185604.csv --pv_map='elementary_school_pvmap.csv,places_resolved.csv' --config_file=elementary_school_metadata.csv --existing_statvar_mcf=gs://unresolved_mcf/scripts/statvar/stat_vars.mcf --output_path=output/elementary_school",
```

Comment:
if the source file has a date suffix that changes everytime the file is downloaded, can we say:
`--input_data=Summary_of_Elementary_School_*.csv`?